src/main/java/seedu/address/model/food/Carbohydrate.java

```diff
@@ -0,0 +1,22 @@
+package seedu.address.model.food;
+
+public class Carbohydrate extends Macronutrient {
```

Comment:
IIRC we need to write javadocs for each class according to the module specifications.